src/emu-i386/simx86/syncpu.h

```diff
@@ -103,10 +103,11 @@ typedef struct {
 /*60*/	unsigned short sigalrm_pending, sigprof_pending;
 /*64*/	unsigned int StackMask;
 /*68*/ 	unsigned int mem_base;
+/*6c*/ 	unsigned int df_increments; /* either 0x040201 or 0xfcfeff */
 	/* begin of cr array */
-/*6c*/	unsigned int cr[5]; /* only cr[0] is used in compiled code */
+/*70*/	unsigned int cr[5]; /* only cr[0] is used in compiled code */
 /* ------------------------------------------------ */
-/*80*/	unsigned int end_mark[0];
+/*80*/	//unsigned int end_mark[0] = cr[4]
```

Comment:
Yes, that's exactly the strategy I am playing with privately.
Trying to change to dword offsets for _some_ fields.
The work looks not too big, although doesn't work for
me so far. Updating the codegen is straight-forward, but
maybe some length tables should be updated too.